content/concepts/tokens/non-fungible-token-transfers.md

```diff
@@ -0,0 +1,82 @@
+---
+html: non-fungible-token-transfers.html
+parent: non-fungible-tokens.html
+blurb: Trading NFTokens in direct or brokered mode.
+filters:
+ - include_code
+labels:
+ - Non-fungible Tokens, NFTs
+status: not_enabled
+---
+
+# Trading NFTokens on the XRP Ledger
+{% include '_snippets/nfts-disclaimer.md' %}
+
+You can transfer NFTokens between accounts on the XRP Ledger. You can transfer `NFTokens` by offering to buy or sell `NFTokens`, or accepting offers from other accounts. You can even give away `NFTokens` by offering to sell them at a price of 0.  All offers are created using the `NFTokenCreateOffer` transaction.
+
+
+## Sell Offers
+
+
+### Create a Sell Offer
+
+As the owner of a `NFToken object`, you can create a sell offer using a `NFTokenCreateOffer` transaction with a _Flags_ setting of _1_. You provide the _NFTokenID_ and the _Amount_ you are willing to accept in payment. You can optionally specify an _Expiration_ date, after which the offer is no longer valid, and a _Destination_ account, which is the only account that is allowed to purchase the `NFToken`.
+
+
+### Accept a Sell Offer
+
+To purchase a `NFToken` that is offered for sale, you use a `NFTokenAcceptOffer` transaction. You provide the owner account and specify the `nft_offer_index` of the `NFTokenOffer` object you choose to accept.
+
+
+## Buy Offers
+
+
+### Create a Buy Offer
+
+Any account can offer to buy a `NFToken` on the XRP Ledger. You can create a buy offer using `NFTokenCreateOffer` with a _Flags_ setting of _null_. You provide the _Owner_ account, _NFTokenID_, and the _Amount_ of your offer.
+
+
+### Accept a Buy Offer
+
+Use the `NFTokenAcceptOffer` transaction to transfer a `NFToken`. Provide the `nft_offer_index` and the owner account to complete the transaction.
+
+
+## Trading Modes
+
+When trading `NFTokens`, you can choose between a _direct_ transaction between a buyer and seller or a _brokered_ transaction, where a third party account matches sell and buy offers to arrange the trade.
+
+Trading in direct mode gives the seller control over the transfer. The seller can either post `NFTokens` for purchase by anyone, or sell NFTokens to a specific destination account. The seller receives the entire purchase price for the `NFToken`.
```

Comment:
There's a non-monospace "NFTokens" in the middle sentence. That should probably be changed for consistency, though per my above comment I'm not sure which way to be consistent is better.